frontend/src/components/buttons/OffSetButtons.jsx

```diff
@@ -0,0 +1,145 @@
+/* eslint-disable react-hooks/exhaustive-deps */
+// eslint-disable-next-line
+import React, { useState, useEffect } from 'react';
+import { useStateValue } from '../../contexts/state';
+import { useHistory, useLocation } from 'react-router-dom';
+import { useQueryParams } from '../../hooks/useQuery';
+
+import {
+    Header,
+    StyledInput,
+    StyledDirectionButton,
+    LatestButton,
+    FlexDiv,
+    StyledEndLeft,
+} from './OffSetButtons.styles';
+
+import { ButtonContainer } from './LastRunCheckbox.styles';
+import Left from '../../images/chevron-left.svg';
+import Right from '../../images/chevron-right.svg';
+const Offset = () => {
+    const history = useHistory();
+    const location = useLocation();
+    const queryParams = useQueryParams();
+
+    const [
+        {
+            offset,
+            parentData: { seriesData },
+        },
+        dispatch,
+    ] = useStateValue();
+    const [inputOffset, setInputOffset] = useState(0);
+    const [leftDisabled, setleftDisabled] = useState(false);
+    const [rightDisabled, setrightDisabled] = useState(false);
+
+    const { builds } = seriesData;
+
+    const updateTags = tag => {
+        queryParams.set('offset', tag);
+        return queryParams.toString();
+    };
+
+    const handleRightButtonPress = () => {
```

Comment:
Nit: Left and right could be the same function with an argument

Reply:
They actually were but the icon used in the button caused the button to function incorrectly, I can try again, now the icon is imported in a different way.